app/backend/hiStoryBackend/api/views/v1/search_views.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework.authtoken.models import Token
+from rest_framework.views import APIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.generics import get_object_or_404
+
+
+from api.helpers import json_response_helpers as jrh
+from api.models import User, MemoryPost
+
+import requests
+from rest_framework.response import Response
+
+
+class SearchView(APIView):
+    
+    def post(self, request):
+        text = request.data.get('text')
+
+        mem_list = []
+        user_list = []
+
+        for post in MemoryPost.objects.all():
+            if text in post.title:
+                if post.id not in mem_list:
+                    mem_list.append(post.id)
+            for story in post.story:
+                if text in story['payload']:
+                    if post.id not in mem_list:
+                        mem_list.append(post.id)
+            if text in post.time['data']:
```

Comment:
Time of a Memory-Post can be empty ('{}`). In this case, `time['data'] == None` and it fails.

Reply:
thanks for the warning, i have added control before accessing.